02 - Community Expansions/02.01 Plugins by Category/Uncategorized plugins.md

```diff
@@ -13,362 +13,362 @@ Plugins which have not yet been categorized by the community.
 
 ## Plugins in this category
 
-- [[13th-age-statblocks|13th Age Statblocks]]: Render 13th Age statblocks in Obsidian.
-- [[obsidian-account-linker|Account Linker]]: Plugin for describing external service accounts in the front matter
-- [[actions-uri|Actions URI]]: Adds additional `x-callback-url` endpoints to the app for common actions — it's a clean, super-charged addition to Obsidian URI.
-- [[obsidian-markdown-file-suffix|Addional Markdown suffix (.mdx/.svx).]]: Use additional files like .mdx / .svx as if they were markdown.
-- [[obsidian-advanced-codeblock|Advanced Codeblock]]: Give additioinal features to  code blocks.
-- [[obsidian-advanced-slides|Advanced Slides]]: Create markdown-based presentations in Obsidian
-- [[obsidian-advanced-new-folder|Advanced new folder]]: This is a plugin that creates a new folder anywhere you want in the vault
-- [[obsidian-agile-task-notes|Agile Task Notes]]: Import your tasks from your TFS (Azure or Jira) to take notes on them and make todo-lists!
-- [[alx-folder-note-folderv|AidenLx's Folder Note - folderv component]]: Optional `folderv` Component for alx-folder-note
-- [[obsidian-alias-from-heading|Alias from heading]]: Implicitly add an alias matching the first heading in a document.
-- [[aosr|Aosr]]: Another obsidian spaced repetition
+- [[recent-files-obsidian|Recent Files]]: List files by most recently opened
```

Comment:
The code for sorting plugins by name has not yet been added to update_releases.py.

This job run didn't add any new plugins, themes or authors, so other than updating theme download stats, it effectively randomised the order of uncategorised plugins.

I'm therefore going to close this PR, un-merged, until I can pair with someone to get update_releases.py to sort plugins when regenerating this file.